packages/esm-patient-chart-app/src/visit/visits-widget/visit-detail-overview.component.tsx

```diff
@@ -80,7 +80,9 @@ function VisitDetailOverviewComponent({ patientUuid }: VisitOverviewComponentPro
                 ))}
               </>
             ) : (
-              <EmptyState headerTitle={t('visits', 'visits')} displayText={t('Visits', 'Visits')} />
+              <div className={styles.emptyStateContainer}>
```

Comment:
This fixes the spacing around the empty state, adding a consistent 1rem vertical margin around the container.